transformers/tokenization_auto.py

```diff
@@ -105,6 +106,8 @@ def from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs):
             return DistilBertTokenizer.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
         elif 'roberta' in pretrained_model_name_or_path:
             return RobertaTokenizer.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
+        elif 'bert-japanese' in pretrained_model_name_or_path:
```

Comment:
Auto models/tokenizer are used with checkpoint names, e.g.: `AutoModel.from_pretrained("bert-base-cased")`

^ it identifies `bert` in the checkpoint name and therefore initializes a Bert model from it.

If I understand correctly all the checkpoints you have have `bert-base-japanese` for prefix, which means that the auto detection won't work. Maybe replace it with `bert-base-japanese`?

Reply:
Thank you for the review, @LysandreJik. I've fixed the AutoTokenizer.